src/nvim/options.lua

```diff
@@ -9915,6 +9915,20 @@ return {
       short_desc = N_('Controls transparency level for floating windows'),
       type = 'number',
     },
+    {
+      abbreviation = 'winbd',
```

Comment:
if the abbreviations are hard to remember they are pretty useless, aren't they? do we actually need an abbreviation for this?

Reply:
`wbd` should be sounds good ?

Reply:
How is that less hard to remember than `winbd`?